File: adminWF/Classes/Wallet.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace auctionComplex.Classes
{
    [Table("wallets")]
    internal class Wallet
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("address"), MaxLength(48)]
        public string Address { get; set; }

        [Column("balance")]
        public double Balance { get; set; }

        public User User { get; set; }
        [Column("user_id")]
        public int UserId { get; set; }
    }
}

File: serverSpring/src/main/java/com/vyatsu/auctionComplex/entity/api/Wallet.java
package com.vyatsu.auctionComplex.entity.api;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.util.LinkedHashSet;
import java.util.Set;

@Entity
@Table(name = "wallets")
public class Wallet {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    @Getter
    @Setter
    private Integer id;

    @Column(name = "address", length = 48)
    @Getter
    @Setter
    private String address;

    @Column(name = "balance", nullable = false)
    @Getter
    @Setter
    private Double balance;

    @ManyToOne(fetch = FetchType.LAZY)
    @JsonIgnore
    @JoinColumn(name = "\"UserId\"")
    @Getter
    @Setter
    private User user;

    @OneToMany(mappedBy = "receiverWallet")
    @JsonIgnore
    @Getter
    @Setter
    private Set<Transaction> receiveTransactions = new LinkedHashSet<>();

    @OneToMany(mappedBy = "senderWallet")
    @JsonIgnore
    @Getter
    @Setter
    private Set<Transaction> sendTransactions = new LinkedHashSet<>();

}